CSS/see_article.css

```diff
@@ -41,50 +46,94 @@ button {
     cursor: pointer;
     font-size: 20px;
 }
-button a {
-    text-decoration: none;
-    font-size: 18px;
-}
-button:hover {
-    background-color: rgb(3, 94, 94);
+
+    button a {
+        text-decoration: none;
+        font-size: 18px;
+    }
+
+    button:hover {
+        background-color: rgb(3, 94, 94);
+    }
+
+    button a:focus {
+        outline: none;
+    }
+
+/*Media querry*/
+
+
+#btn_add_article_to_cart {
+    padding: 4px;
+    background-color: rgb(220, 150, 220);
+    border: none;
+    cursor: pointer;
+    width: 100%;
 }
-button a:focus {
+
+/*All the css used to customize the input of type : number*/
+.number-input button {
     outline: none;
+    -webkit-appearance: none;
+    background-color: transparent;
+    border: none;
+    align-items: center;
+    justify-content: center;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    margin: 0;
+    position: relative;
+    box-shadow: 0px 0px 1px #474747;
 }
 
-/*Media querry*/
-@media(max-width: 1300px) {
-    button {
-        padding: 8px 13px;
-        background-color: rgb(3, 122, 122);
-        border: none;
-        border-radius: 5px;
-        cursor: pointer;
-    }
+.number-input input[type=number] {
+    font-family: sans-serif;
+    padding: .36px;
+    border: none;
+    border-width: 0 2px;
+    font-size: 17px;
+    height: 20px;
+    font-weight: bold;
+    text-align: center;
+    color: #9be3df;
 }
 
-@media(max-width: 1170px) {
-    button a {
-        font-size: 15px;
+
+@media(max-width: 650px) {
```

Comment:
this block will always override the defaults, and contains duplicated code :)

Reply:
Deleted